pkg/common/cns-lib/volume/listview.go

```diff
@@ -317,3 +317,15 @@ func (l *ListViewImpl) MarkTaskForDeletion(ctx context.Context, taskMoRef types.
 	log.Infof("%v marked for deletion", taskMoRef)
 	return nil
 }
+
+// LogoutSession is a setter method to logout vcenter session created
+func (l *ListViewImpl) LogoutSession(ctx context.Context) error {
+	log := logger.GetLogger(ctx)
+	log.Info("logout list view vCenter session")
```

Comment:
print vc host name in the log

Reply:
made changes as suggested in latest commit